### PR TITLE
More hyperv events for electron

### PR DESCRIPTION
**Pull request by maaktweluit**
- [x] SMB port blocked
- [x] Not enough disk space
- [x] Vt-x not enabled
- [x] Saving / resuming vm

### CODE REVIEW

**Review by Wiezzel — APPROVED** [review]

File: golem/docker/hypervisor/hyperv.py [hunk 1]
```diff
@@ -20,18 +20,36 @@
 from golem.docker.config import CONSTRAINT_KEYS, MIN_CONSTRAINTS
 from golem.docker.hypervisor.docker_machine import DockerMachineHypervisor
 from golem.docker.task_thread import DockerBind
-from golem.report import Component, Stage
+from golem.report import Component, Stage, report_calls
 from golem.rpc.mapping.rpceventnames import Golem
 
 logger = logging.getLogger(__name__)
 
+EVENT_SMB = 'smb_blocked'
```
Comment (Wiezzel) [inline golem/docker/hypervisor/hyperv.py]: Looks like a job for enum to me.

File: golem/docker/hypervisor/hyperv.py [hunk 2]
```diff
@@ -394,12 +411,16 @@ def _get_max_memory(self, constr: Optional[dict] = None) -> int:
         return hardware.pad_memory(int(0.9 * max_mem_in_mb))
 
     @staticmethod
-    def _lowered_memory_event(mem_mb: int) -> None:
-        message = MESSAGES['lowered_memory'].format(mem_mb=mem_mb)
-        event = EVENTS['lowered_memory'].copy()
+    def _handle_event(name, **kwargs) -> None:
```
Comment (Wiezzel) [inline golem/docker/hypervisor/hyperv.py]: More like 'publish_event'

**Review by Wiezzel — APPROVED** [review]
Two minor code-style comments. Feel free to ignore if you don't want to change things.

**Review by mfranciszkiewicz — APPROVED** [review]
What @Wiezzel said

File: golem/docker/hypervisor/hyperv.py (hunk 1 above)
Comment (maaktweluit) [reply golem/docker/hypervisor/hyperv.py]: good one, will look into this

File: golem/docker/hypervisor/hyperv.py (hunk 2 above)
Comment (maaktweluit) [reply golem/docker/hypervisor/hyperv.py]: there is already a function in the same file called `publish_event` this one is more `log_and_publish_event`

do you think that is a better name?

Comment (Wiezzel) [reply golem/docker/hypervisor/hyperv.py]: Well, it will do.

**Comment by maaktweluit** [comment]
@Wiezzel both comments are resolved, could you check again?

**Comment by Wiezzel** [comment]
@maaktweluit It's fine. 👍

**Pull request merged**